words/objects.txt

```diff
@@ -1601,14 +2069,22 @@ police
 policeman
 polish
 politician
+polka
 polo
+polonium
+polyanthus
 polyester
+pomegranate
+pomelo
 pond
 pony
+popa
```

Comment:
what is this?

Reply:
it means "butt" in Russian http://masterrussian.net/f31/whats-means-pipiska-popa-9849/

Reply:
whelp, whatever it means in english, that's a good enough reason to take it out!